components/layout/sidebar.tsx

```
@@ -29,107 +33,174 @@ export function Sidebar() {
+
+        {/* Divider */}
+        <div className="my-3 mx-4 border-t border-white/[0.06]" />
+
+        {/* Settings & Feedback */}
+        <div className="space-y-0.5 px-2">
+          <button
             onClick={() => { openRightPanel('settings'); closeSidebarOnMobile(); }}
+            title={sidebarCollapsed ? 'Settings' : undefined}
+            className={`w-full flex items-center rounded-xl transition-all duration-150 text-white/40 hover:text-white hover:bg-white/[0.06]
+              ${sidebarCollapsed ? 'justify-center h-10 w-10 mx-auto' : 'gap-3 px-3 py-2.5'}`}
           >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)